windows/src/engine/keyman32/kmhook_getmessage.cpp

```diff
@@ -412,6 +420,32 @@ void ProcessWMKeymanControlInternal(HWND hwnd, WPARAM wParam, LPARAM lParam)
 	}
 }
 
+void
+ProcessWMKeymanControl(WPARAM wParam, LPARAM lParam) {
+
+  switch (wParam) {
+  case KMC_PROFILECHANGED:
+    if (!Globals::IsControllerThread(GetCurrentThreadId())) {
+      break;
+    }
+    WORD wAtom = HIWORD(lParam);
+    char atomStr[128];
```

Comment:
Can we move these outside the `switch` statement as their scope is rubbery otherwise...